tools/mapmerge2/requirements.txt

```diff
@@ -1,3 +1,3 @@
-pygit2==0.27.2
+pygit2==1.1.1
 bidict==0.13.1
-Pillow==6.2.0
+Pillow==7.0.0
```

Comment:
When you told me you were gonna bump up the mapmerge requirements, I thought you were going to do that in a separate PR (hell even a webedit), you lazy.

Reply:
...I literally asked if I could include this in my pr and you said yes